app/scripts/abos/detail/abosdetail.html

```diff
@@ -45,7 +45,7 @@ <h3 class="panel-title"><span class="navbar-brand noLeftSpace"><span translate>A
         <div class="col-md-4">
           <label translate>Vertriebsart</label>
         </div>
-        <div class="col-md-8">{{vertriebsart()}} - {{abo.vertrieb.liefertag}}</div>
+        <div class="col-md-8">{{vertriebsart() |translate}} - {{abo.vertrieb.liefertag |translate}}</div>
```

Comment:
Vertriebsart ist in Stammdaten definiert (pro Projekt) und kann nicht global übersetzt werden.